.gitignore

```diff
@@ -2,6 +2,8 @@ build*/
 winbuild
 win64build
 .vs/
+CMakeCache.txt
+CMakeFiles/
```

Comment:
These two `CMake*` entries don't seem necessary. It shouldn't be built in the source tree like that, and there's a number of other files it would create there if it was.

Reply:
updated!